src/System.Management.Automation/engine/MshObject.cs

```diff
@@ -630,31 +629,6 @@ internal static PSObject ConstructPSObjectFromSerializationInfo(SerializationInf
 
         private PSObjectFlags _flags;
 
-        /// <summary>
-        /// This field contains a stream type used by the formatting system.
-        /// </summary>
-        private WriteStreamType _writeStream;
-
-        /// <summary>
-        /// Members from the adapter of the object before it was serialized
-        /// Null for live objects but not null for deserialized objects.
-        /// </summary>
-        private PSMemberInfoInternalCollection<PSPropertyInfo> _adaptedMembers;
-
-        /// <summary>
-        /// Members from the adapter of the object before it was serialized
-        /// Null for live objects but not null for deserialized objects.
-        /// </summary>
-        private PSMemberInfoInternalCollection<PSPropertyInfo> _clrMembers;
-
-        // This is toString value set on deserialization
-        private string _toStringFromDeserialization;
-
-        /// <summary>
-        /// If this is non-null return this string as the ToString() for this wrapped object.
-        /// </summary>
-        private string _tokenText;
```

Comment:
These private fields are not necessary. We can just use automatic properties directly.

Reply:
I have a PR in the pipeline for extracting those to another ocject. They are rarely used.